docs/experimentsetup.rst

```diff
@@ -14,8 +14,10 @@ Electrochemical cells/devices commonly have two or three leads. An important dis
 dependencies of the potential.
 
 In a three electrode cell, there is a working electrode (with the material or reaction of interest) and counter electrode
-(completes the circuit and is a source or sink for electrons by facilitating a counter reaction) and a reference electrode.
-The reference electrode does not undergo any reactions, and it maintains a constant potential.
+(completes the circuit and allows charge to flow by facilitating a counter reaction) and a reference electrode.
+The reference electrode does not undergo any reactions, and it maintains a constant potential. The standard hydrogen
+electrode (SHE) is based on the formation of hydrogen gas from two protons and two electrons. The potential at which this
```

Comment:
On a Pt electrode in an ideal solution as well

Reply:
Looking good!

Reply:
Sounds good, I wasn't sure how much information was necessary without needing to go deep into a discussion about reference electrodes in general. I think with this addition it should be sufficient.

Reply:
👍